scripts/qa_crowbarsetup.sh

```diff
@@ -2984,20 +2984,21 @@ function oncontroller_tempest_legacy()
     wait_for 300 5 \
         'openstack image show $imageid | grep active &>/dev/null' \
         "prepare cnftools image"
+
     pushd /var/lib/openstack-tempest-test
-    echo 1 > /proc/sys/kernel/sysrq
-    if iscloudver 5plus; then
-        if tempest help cleanup; then
-            tempest cleanup --init-saved-state
-        else
-            /usr/bin/tempest-cleanup --init-saved-state || :
+        echo 1 > /proc/sys/kernel/sysrq
```

Comment:
this indentation is not used consistently in the rest of the code, maybe we can avoid doing this here and do that in a different PR that address other cases